Cargo.lock

```diff
@@ -1510,7 +1579,7 @@ checksum = "ec3efd23720e2049821a693cbc7e65ea87c72f1c58ff2f9522ff332b1491e590"
 dependencies = [
  "futures-util",
  "http 0.2.12",
- "hyper",
```

Comment:
unfortunately we have both hyper 1.0 and hyper 0.14 now

Reply:
This will not be a problem after we upgrading reqwest to latest.